mm2src/coins/nft_storage/mod.rs

```diff
@@ -0,0 +1,159 @@
+use crate::nft::nft_structs::{Chain, Nft, NftList, NftTokenAddrId, NftTransferHistory, NftTxHistoryFilters,
+                              NftsTransferHistoryList, TxMeta};
+use async_trait::async_trait;
+use derive_more::Display;
+use mm2_core::mm_ctx::MmArc;
+use mm2_err_handle::mm_error::MmResult;
+use mm2_err_handle::mm_error::{NotEqual, NotMmError};
+use mm2_number::BigDecimal;
+use serde::{Deserialize, Serialize};
+use std::num::NonZeroUsize;
+
+#[cfg(not(target_arch = "wasm32"))] pub mod sql_storage;
+#[cfg(target_arch = "wasm32")] pub mod wasm_storage;
+
+#[derive(Debug)]
+pub enum RemoveNftResult {
+    NftRemoved,
+    NftDidNotExist,
+}
+
+pub trait NftStorageError: std::fmt::Debug + NotMmError + NotEqual + Send {}
+
+#[async_trait]
+pub trait NftListStorageOps {
```

Comment:
It's good to add unit tests for `NftListStorageOps` for both native and wasm similar to how it was done in `utxo_block_header_storage` https://github.com/KomodoPlatform/atomicDEX-API/blob/1c261cd5a39694ae13f2699a5c4761df3468e19d/mm2src/coins/utxo/utxo_block_header_storage/mod.rs#L116 This will prevent regression if there will be any changes to DB structure or methods in the future. It will also make sure that the implementation for wasm works the same way as the sqlite one. Please consider this in the next PRs.